javascript-bank/bank.js

```diff
@@ -13,7 +13,7 @@ function Bank() {
 Bank.prototype.openAccount = function (holder, balance) {
   if (Number.isInteger(balance) === true && balance > 0) {
     var newAccount = new Account(this.nextAccountNumber, holder);
-    Account.getBalance += newAccount.deposit(balance);
+    Account.getBalance = newAccount.deposit(balance);
```

Comment:
This is also incorrect. Is this passing all tests? You didn't tell me with your last update.

Reply:
You should change this line to just `newAccount.deposit(balance);`